modules/machineset-yaml-osp.adoc

```diff
@@ -66,6 +66,7 @@ spec:
           trunk: true
           userDataSecret:
             name: <node_role>-user-data <2>
+          az: <optional_list_of_openstack_availability_zones>
```

Comment:
I think this is `availabilityZone`. Did you find `az` in our upstream docs?

Reply:
No. I'm going to sub in availabilityZone for now.